### ISSUE TITLE
Official ExRole Chinese Language Support

### ISSUE THREAD

**Issue by yukieiji** [FFxFc/ExtremeRolesX#1]
Hi, I am yukieiji, an ExRole developer.
Since EXRoles is multilingual, shall I officially support it?

Since another repository or fork may be in violation of hats or nameplates license (in fact, the nameplates I created bans its use on forks, etc.), it is safer to default support Chinese.

**Comment by fivefirex** [comment]
Yeah you certainly can. I will keep following up with you for updates afterwards.
Should I help you to merge the json file?

**Comment by fivefirex** [comment]
I'm doing my best to merge the text, but the other members of my group are doing other things, so this may take a longer time, sorry very much. 😢

**Comment by yukieiji** [comment]
Okay, thank you.
No need to rush, please work at your own pace.
